src/main/java/com/playmonumenta/networkrelay/BroadcastCommand.java

```diff
@@ -14,62 +19,95 @@
 
 	private final Logger mLogger;
 
 	protected BroadcastCommand(Plugin plugin) {
 		mLogger = plugin.getLogger();
 
-		CommandAPICommand innerCommand = new CommandAPICommand("broadcastcommand")
+		CommandAPICommand broadcastCommand = new CommandAPICommand("broadcastcommand")
 			.withPermission(CommandPermission.fromString("monumenta.networkrelay.broadcastcommand"))
 			.withArguments(new GreedyStringArgument("command"))
 			.executes((sender, args) -> {
-				run(plugin, sender, (String)args[0]);
+				run(plugin, sender, (String)args[0], null);
+			});
+
+		CommandAPICommand broadcastBungeeCommand = new CommandAPICommand("broadcastbungeecommand")
+			.withPermission(CommandPermission.fromString("monumenta.networkrelay.broadcastbungeecommand"))
+			.withArguments(new GreedyStringArgument("command"))
+			.executes((sender, args) -> {
+				run(plugin, sender, (String)args[0], NetworkRelayAPI.ServerType.BUNGEE);
+			});
+
+		CommandAPICommand broadcastMinecraftCommand = new CommandAPICommand("broadcastminecraftcommand")
+			.withPermission(CommandPermission.fromString("monumenta.networkrelay.broadcastminecraftcommand"))
+			.withArguments(new GreedyStringArgument("command"))
+			.executes((sender, args) -> {
+				run(plugin, sender, (String)args[0], NetworkRelayAPI.ServerType.MINECRAFT);
 			});
 
 		// Register first under the monumenta -> networkRelay namespace
 		new CommandAPICommand("monumenta")
 			.withSubcommand(new CommandAPICommand("networkRelay")
-				.withSubcommand(innerCommand)
+				.withSubcommand(broadcastCommand)
+				.withSubcommand(broadcastBungeeCommand)
+				.withSubcommand(broadcastMinecraftCommand)
 			).register();
 
 		// Then directly, for convenience
-		innerCommand.register();
+		broadcastCommand.register();
+		broadcastBungeeCommand.register();
+		broadcastMinecraftCommand.register();
 	}
 
-	private static void run(Plugin plugin, CommandSender sender, String command) {
+	private static void run(Plugin plugin, CommandSender sender, String command, NetworkRelayAPI.ServerType serverType) {
 		if (!ENABLED) {
 			sender.sendMessage("This command is not enabled");
 		}
 
 		/* Get the player's name, if any */
 		String name = "";
 		if (sender instanceof Player) {
-			name = ((Player)sender).getName();
+			name = sender.getName();
 		} else if (sender instanceof ProxiedCommandSender) {
 			CommandSender callee = ((ProxiedCommandSender) sender).getCallee();
 			if (callee instanceof Player) {
-				name = ((Player)callee).getName();
+				name = callee.getName();
 			}
 		}
 
 		/* Replace all instances of @S with the player's name */
 		command = command.replaceAll("@S", name);
 
-		if (!(sender instanceof Player) || ((Player)sender).isOp()) {
-			sender.sendMessage(ChatColor.GOLD + "Broadcasting command '" + command + "' to all servers");
+		String typeStr;
+		switch (serverType) {
+			case BUNGEE:
+				typeStr = "all bungee";
+				break;
+			case MINECRAFT:
+				typeStr = "all minecraft";
+				break;
+			default:
```

Comment:
Good practice to still have case
```
ALL:
default: 
```

So the all case is explicit, and if theoretically we ever have a tool to catch that all enums are handled it'd be set up.